src/main/java/com/tstepnik/planner/service/UserService.java

```diff
@@ -25,13 +25,14 @@ public UserService(UserRepository userRepository) {
         return userRepository.findAll();
     }
 
+    //TODO change logic, don;t use database for logged user
     public User getLoggedUser(Principal principal) {
```

Comment:
Where is this method used?

Reply:
I don't understand question, don't you see where it is? I know that i should change develop so I just add TODO for feauture.

Reply:
Since i have logged user from SecurityContextHolder i don't need this method so I'm going to remove it.

Reply:
Exactly, thats what I meant.